Aztec/Classes/Public/TextKit/AztecTextAttachment.swift

```diff
@@ -40,22 +35,80 @@ public class AztecTextAttachment: NSTextAttachment
         super.init(coder: aDecoder)
     }
 
+    // MARK: - Origin calculation
+
+    func xPosition(forContainerWidth containerWidth: CGFloat) -> Int {
+
```

Comment:
That said, I'll remove it from this particular method, since its not really needed.